docs/modules/ROOT/pages/capabilities-explained.adoc

```diff
@@ -73,7 +93,7 @@ The table below highlights the differences between the two key options.
 
 |===================================
 
-* when request is made to get features from FH server, it can, for example, be visible from the client's browser Dev tools, including the request url, that consists of environment id and API Key. As Server evaluated key doesn't return any sensitive information, it doesn't need to be treated as "secret". On contrary, Client evaluated keys should be kept "secret" because they return the data that contains all the features and rollout strategy information which can be sensitive (e.g. user id's, emails, etc.)
+(*) when request is made to get features from FH server, it can, for example, be visible from the client's browser Dev tools, including the request url, that consists of environment id and API Key. As Server evaluated key doesn't return any sensitive information, it doesn't need to be treated as "secret". On contrary, Client evaluated keys should be kept "secret" because they return the data that contains all the features and rollout strategy information which can be sensitive (e.g. user id's, emails, etc.)
```

Comment:
just slight wording "when a request is made", rather than "when request is made"